src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,19 @@
 package core.basesyntax;
 
+import java.util.Random;
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
+    private static final int COLOR_INDEX_BOUND = 6;
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        return COLORS[new Random().nextInt(COLOR_INDEX_BOUND)];
```

Comment:
Are you sure that your  random can give you each index?and you can use COLORS.length instead of another constant